### ISSUE TITLE
Could you please consider publishing this to the Open VSX Registry?

### ISSUE THREAD

**Issue by Arkanic** [Antyos/vscode-mlog#2]
https://open-vsx.org/

This would allow support for other editors that use vscode-style extensions, but don't use the visual studio marketplace.

Thanks for the great extension BTW.

**Comment by Antyos** [comment]
My other VS Code extension is on OpenVSX but I guess I forgot that this one wasn't... oops! 😅

I added a workflow that should auto-publish the extension to the OpenVSX marketplace and I also added my extension to [open-vsx/publish-extensions](https://github.com/open-vsx/publish-extensions). (See [open-vsx/publish-extensions#366](https://github.com/open-vsx/publish-extensions/pull/366)).

The extension should hopefully be published within the next 24 hours. Let me know if you have any other issues!
